app/routes/story.routes.js

```diff
@@ -0,0 +1,26 @@
+module.exports = (app) => {
+    const Story = require("../controllers/story.controller.js");
+    const { authenticateRoute } = require("../authentication/authentication");
+    var router = require("express").Router();
+  
+    // Create a new Recipe
+    router.post("/stories/", [authenticateRoute], Story.create);
```

Comment:
If you pull from main use get the stories.route.js file, you can omit this method, I already have it defined.

Reply:
don't pull `main` directly into your B_08218 branch, nor merge from `main` to B_08218, though :)
instead, checkout `main`, pull the latest commits, then checkout your B_08218 branch and `git rebase main` to rebase your branch as if it were starting from the "latest" `main` branch

Reply:
> don't pull `main` directly into your B_08218 branch, nor merge from `main` to B_08218, though :) instead, checkout `main`, pull the latest commits, then checkout your B_08218 branch and `git rebase main` to rebase your branch as if it were starting from the "latest" `main` branch

This is true. When I say "pull from main", I should define this process as what I mean.